tests/pbh_test.py

```diff
@@ -919,7 +999,8 @@ def test_show_pbh_statistics_after_disabling_rule(self):
 
         result = runner.invoke(
             config.config.commands["pbh"].
-            commands["rule"].commands["update"],
+            commands["rule"].commands["update"].
+            commands["field"].commands["set"],
```

Comment:
Just curious, what is the purpose of this additional line compared to older version?

Reply:
@kktheballer has been already discussed